src/main/proto/user_service.proto

```diff
@@ -41,7 +59,36 @@ message CoopResponse {
     string logo = 4;
 }
 
+message GetActiveUsersRequest {
+    string uuid = 1;
+    string coop = 2;
+}
+
+message UserExtendedResponse {
+    string uuid = 1;
+    string first_name = 2;
+    string last_name = 3;
+    uint64 createdAt = 4;
+    string language = 5;
+    string dateOfBirth = 6;
+    string documentNumber = 7;
+    string dateOfIssue = 8;
+    string dateOfExpiry = 9;
+    string personalNumber = 10;
+    Role role = 11;
```

Comment:
I think `role` won't be needed when access validation is done in the controller via JWT.